app/src/main/java/org/greenstand/android/TreeTracker/dashboard/components/DisplayAlertDialog.kt

```diff
@@ -0,0 +1,48 @@
+package org.greenstand.android.TreeTracker.dashboard.components
+
+import androidx.compose.material.AlertDialog
+import androidx.compose.material.Button
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun DisplayAlertDialog(
```

Comment:
We have a dialog method we can reuse that has a special style that matches the app. Please use that instead